src/labels/CustomLabels.labels

```diff
@@ -1583,6 +1607,14 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <protected>true</protected>
         <shortDescription>The RD2 Batch job can only be executed when Enhanced RD is enabled</shortDescription>
         <value>You must enable Enhanced Recurring Donations before you can run the Enhanced Recurring Donations batch job.</value>
+    </labels>
+     <labels>
+        <fullName>RD2_InstallmentPeriodMustBeValid</fullName>
+        <categories>Recurring-Donations, Error</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Validate Recurring Donations data if it is valid install period</shortDescription>
```

Comment:
@sam-knox , can you please verify new Custom Labels' short description and value?